src/Orleans.Core/Configuration/ServiceCollectionExtensions.cs

```diff
@@ -124,7 +124,7 @@ public static IServiceCollection AddIncomingGrainCallFilter(this IServiceCollect
         /// <param name="services">The service collection.</param>
         /// <param name="filter">The filter.</param>
         /// <returns>The service collection.</returns>
-        public static IServiceCollection AddOutgoingGrainCallFilter(this IServiceCollection services, IOutgoingGrainCallFilter filter)
+        internal static IServiceCollection AddOutgoingGrainCallFilter(this IServiceCollection services, IOutgoingGrainCallFilter filter)
```

Comment:
why make those methods internal? as I understand, we make provider extension methods on IServiceCollection because they need to configure ApplicationPart and IServiceCollection doesn't have access to ApplicationPart. But this is not true with GrainCallFilters , right?

Reply:
We haven't released this code as stable yet, so I rather make them internal while there are no consumers to break.